proto/validator/accounts/v2/web_api.proto

```diff
@@ -200,6 +206,15 @@ message WalletResponse {
     KeymanagerKind keymanager_kind = 2;
 }
 
+message RecoverWalletRequest {
+    string mnemonic = 1;
+    uint64 num_accounts = 2;
+    string wallet_password = 3;
+    string language = 4;
+    string mnemonic25th_word =5 ;
+    bool   skip_mnemonic25th_word = 6; 
```

Comment:
a bool here seems redundant. If the request has `mnemonic25th_word == '' ` , it is a reasonable assumption that the requester is skipping the 25th word.

Reply:
So Web UI handles whether to skip or not and if not passes the phrase. Backend rpc only worries to check that it is not an empty string (strings.TrimSpace(mnemonic) != '') ?

Reply:
That is correct, thats how it looks like from my point of view. I am not privy to any previous discussions @rauljordan might have had with you on this regarding the schema but it seems like we can be more efficient here.

Reply:
We haven't implemented the 25th word feature on the frontend, so I think it is ok for it to be optional. The backend just has to check it is an empty string, so I would get rid of this bool in the request